engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -181,7 +186,21 @@ private List<NotificationHistory> sendEmail(Event event, Set<Endpoint> endpoints
         }
 
         if (subject == null || body == null) {
-            return Collections.emptyList();
+            if (subject != null || body != null) {
+                Log.warnf("Only one of the subject and body was found for event: %s/%s/%s", event.getBundleDisplayName(), event.getApplicationDisplayName(), event.getEventTypeDisplayName());
+            }
+
+            if (featureFlipper.isUseDefaultTemplate()) {
+                if (subject == null) {
+                    subject = Default.getTitle();
+                }
+
+                if (body == null) {
+                    body = Default.getBody();
+                }
```

Comment:
I'm not sure about this.

Mixing a default title with a non-default body could go unnoticed from an onboarded app perspective.

Also, mixing a default body with a non-default title will result in an email notification with a message that is not accurate: there is actually an email template associated with the event type, but one part of the template is missing.

Reply:
You are right there, I guess we could overwrite both and put a message in the email body

"This event type had a subject (<or body>) template, but not <the other>"
The warning would be useful too.